week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -7,8 +7,9 @@
   NOTE: only the names should be returned, not the means of transport.
 */
 
-function journeyPlanner() {
-
```

Comment:
Looks like two of the tests are still failing for this one. Any ideas why that might be?